lms/templates/learner_dashboard/program_details_fragment.html

```diff
@@ -13,11 +13,12 @@
 <%block name="js_extra">
 <%static:webpack entry="ProgramDetailsFactory">
 ProgramDetailsFactory({
-    programData: ${program_data | n, dump_js_escaped_json},
+    programData: ${program_data | n,dump_js_escaped_json},
```

Comment:
This whitespase might has been deleted by mistake.

Reply:
+1